tests/integration/test_requirements_validation.py

```
@@ -0,0 +1,358 @@
+        for i, line in enumerate(requirements_lines, 1):
+            if not line.startswith('#'):
+                assert 'http://' not in line, \
+                    f"Line {i}: Use https:// instead of http://"
+    
+    def test_no_spaces_around_operators(self, requirements_lines):
+        """Verify operators have no surrounding spaces."""
+        for i, line in enumerate(requirements_lines, 1):
+            if not line.startswith('#'):
+                # Check for spaces around version operators
+                assert ' == ' not in line and ' >= ' not in line and ' <= ' not in line, \
+                    f"Line {i}: Remove spaces around operators: {line}"
```

Ruff (0.14.7)

Assertion should be broken down into multiple parts

(PT018)